src/main/java/be/seeseemelk/mockbukkit/entity/ArmorStandMock.java

```diff
@@ -386,85 +395,97 @@ public boolean isSlotDisabled(@NotNull EquipmentSlot slot)
 	@Override
 	public @NotNull Rotations getBodyRotations()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return toPaper(getBodyPose());
 	}
 
 	@Override
 	public void setBodyRotations(@NotNull Rotations rotations)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		Preconditions.checkNotNull(rotations, "Rotations cannot be null");
+		setBodyPose(toBukkit(rotations));
```

Comment:
This was based on the Paper implementation. In paper implementation, both "setBodyRotations" and "setBodyPose" share the same variable.